src/funcCoreTools/installOrUpdateFuncCoreTools.ts

```diff
@@ -31,12 +31,11 @@ export async function installOrUpdateFuncCoreTools(): Promise<void> {
             packageManager = (await ext.ui.showQuickPick(picks, { placeHolder })).data;
         }
 
-        let projectRuntime: ProjectRuntime | undefined = await tryGetLocalRuntimeVersion();
-        // tslint:disable-next-line:strict-boolean-expressions
-        if (!projectRuntime) {
-            projectRuntime = await promptForProjectRuntime(localize('selectLocalRuntime', 'Failed to detect local runtime automatically. Select your runtime to update'));
+        let version: FuncVersion | undefined = await tryGetLocalFuncVersion();
+        if (version === undefined) {
```

Comment:
Why break style all of a sudden and write `version === undefined`?

Reply:
I use `=== undefined` somewhat frequently for enums, so I don't think it's breaking style.